src/controls/sitePicker/SitePicker.tsx

```diff
@@ -67,7 +67,11 @@ const styles = mergeStyleSets({
   },
 });
 
-const async = new Async();
+useEffect(() => {
```

Comment:
with this change `async` is not initialized anymore so it will be `undefined`.
Could you please update the code to return back initialization (maybe as a memo or ref)?

Thank you!

Reply:
My mistake since this was in eval scope. `useAsync` will memoize and dispose properly within FC scope - https://github.com/microsoft/fluentui/blob/c28f7314c8a967a0f6f7bd6e4a7df9a5ab7703fc/packages/react-hooks/src/useAsync.ts#L7. I updated the branch, thanks.